Resources/Prototypes/_Goobstation/tags.yml

```
@@ -582,3 +585,6 @@
 
 - type: Tag
   id: MidroundAntag
+
+- type: Tag
+  id: LingEdible
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)